### PR TITLE
removes copyright infringement from the clown name lists

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

File: config/names/clown.txt [hunk 1]
```diff
@@ -26,12 +26,11 @@ Pocket
 Razzle Dazzle
 Redshirt McBeat
 Ronnie Pace
-Scootaloo
```
Comment (MrStonedOne) [inline config/names/clown.txt]: wot?

Comment (Shadowlight213) [inline config/names/clown.txt]: Lmao
It's a my little pony name.

Comment (optimumtact) [inline config/names/clown.txt]: It's an MLP pony but I can't find any data indicating the name is actually copyright or trademarked.

**Comment by Shadowlight213** [comment]
#gooftriggered

**Comment by ExcessiveUseOfCobblestone** [comment]
bronies btfo'd

**Comment by Nienhaus** [comment]
@optimumtact does this count? http://www.trademarkia.com/scootaloo-76689676.html Wasn't to hard to Google.

**Comment by evsey9** [comment]
lmao

**Comment by optimumtact** [comment]
I suppose it does

**Comment by Iamgoofball** [comment]
@excessive nah I just played comp tf2 and ran into a lot of creepers talking about their favorite pastel horses they want to rape

**Comment by Shadowlight213** [comment]
@optimumtact it doesn't
The trademark is in relation to "toy ponies and accessories for use therewith"
So it's for toy ponies and toy pony accessories

**Comment by Iamgoofball** [comment]
ss13 is a toy

**Comment by Shadowlight213** [comment]
@Iamgoofball is it a toy PONY though?

**Comment by A-t48** [comment]
Please see #20362, it alleviates this situation.

**Comment by MrStonedOne** [comment]
a trademark doesn't apply here because a clown is not a pony.

Trademarks are very narrow in application. That trademark only applies to toy ponies, nothing more.

**Comment by PKPenguin321** [comment]
Instead of outright removing the name replace it with "Big Slappy"

**Comment by Supermichael777** [comment]
See the thing about trademark is that it's specific. because we aren't even selling toys, much less horse themed ones, we're unlikely to run into problems

**Comment by Iamgoofball** [comment]
Yeah I don't care it's getting removed

**Comment by InsaneHyena** [comment]
Bronies BTFO

**Comment by FlavoredCactus** [comment]
I second PKPenguin, bring on big slappy

**Comment by Shadowlight213** [comment]
Then the pr and commit titles are misleading and false.
It should be changed to "Removes clown name that's triggers goofball"

**Comment by NocturnalQuill** [comment]
If copyright is a serious concern, our usage of these names could easily qualify as fair use.

**Comment by Shadowlight213** [comment]
It's not a concern. Goof is just using it as an excuse. I should go remove freon for being copyrighted.

**Comment by optimumtact** [comment]
Alright, lets not get carried away.

**Pull request closed without merging**